include/eve/memory/pointer.hpp

```diff
@@ -9,6 +9,7 @@
 
 #include <eve/memory/aligned_ptr.hpp>
 #include <eve/detail/kumi.hpp>
+#include <eve/concept/range.hpp>
```

Comment:
why ?

Reply:
Is is used in the header in question and was missing in some include path.